ml-conversational-analytic-tool/tests/test_commentAnalysis.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from unittest.mock import MagicMock
+
+import commentAnalysis
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    def setUp(self) -> None:
+        global sentiment_analyzer
+        sentiment_analyzer = commentAnalysis.SentimentIntensityAnalyzer()
+        sentiment_analyzer.polarity_scores = MagicMock(return_value={"compound": 1.0})
```

Comment:
I believe global is not really a best practice, instead you can use self
```suggestion
        self.sentiment_analyzer = commentAnalysis.SentimentIntensityAnalyzer()
        self.sentiment_analyzer.polarity_scores = MagicMock(return_value={"compound": 1.0})
```

this would mean that you have to add `self` to every test that uses sentiment_analyzer